### DISCUSSION TITLE
Unable to override file from Upload plugin

### DISCUSSION THREAD

**Discussion by ramlez** [strapi/strapi#6122] [Legacy Topics]
Hello guys, 

**My goal**
* extend `strapi-plugin-upload` File's model (**DONE**) 
* display new fields during file edition (on the modal window in Media Library) (**Not working**) 

I've successfully extended the File's model. New fields are visible in the database after adding new files. However, I'm not able to display these new fields on File's edit form (in Media Lib)

I tried to override this file: `/strapi-plugin-upload/admin/src/components/EditForm/utils/form.js` 

In my project: `<my-project>/extensions/upload/admin/src/components/EditForm/utils/form.js`

But it doesn't work. I'm not sure if I'm doing something wrong or is it a bug? :) 

Thank you in advance.

**Comment by MattieBelt** [comment]
@ramlez That should work, did you rebuild you admin panel? (yarn build --clean)

**Reply by ramlez** [reply]
Thank you @MattieBelt. That's helped and now it works. 
Do you know how to achieve it while developing the admin panel on port `4000`?

**Reply by MattieBelt** [reply]
@ramlez `yarn develop --watch-admin` works for customizing the admin panel, you could try that.
I'm only not sure if it works for plugin admin extension. (I did not try it)

**Comment by weidox** [comment]
I tried to override just model in /extensions/upload/models/File.settings.json, I could not get even a new field to work, wasn't being returned from API responses when I tried to fill it during upload.
